include/exadg/incompressible_navier_stokes/user_interface/enum_types.h

```diff
@@ -638,6 +652,43 @@ std::string
 enum_to_string(SchurComplementPreconditioner const enum_type);
 
 
+
+/**************************************************************************************/
+/*                                                                                    */
+/*                            SOLVE MASS SYSTEM HDIV                                  */
+/*                                                                                    */
+/**************************************************************************************/
+
+/*
+ *  Solver type for solution of mass system for hdiv:
+ *
+ *  We cannot apply the inverse mass operator while using HDIV. Instead we have to solve
+ *  the system
+ *
+ */
+enum class SolverMassHdiv
+{
+  CG
+};
+
+std::string
+enum_to_string(SolverMassHdiv const enum_type);
```

Comment:
I think we do not need this as long as there is not decision to be made. Let's try to minimize the parameters seen by a user.

Reply:
I wasn't even checking it when initiating the solver :D